src/ir/auth_logic/ast.h

```diff
@@ -63,9 +63,10 @@ class Predicate {
 class Attribute {
  public:
   explicit Attribute(Principal principal, Predicate predicate)
-    : principal_(principal), predicate_(predicate) {}
+      : principal_(principal), predicate_(predicate) {}
```

Comment:
`std::move` from parameters to field initializers.

Reply:
done.